includes/Core/Key_Metrics/Key_Metrics_Settings.php

```diff
@@ -49,7 +49,7 @@ protected function get_default() {
 		return array(
 			'widgetSlugs'                      => array(),
 			'isWidgetHidden'                   => false,
-			'includeConversionTailoredMetrics' => false,
+			'includeConversionTailoredMetrics' => array(),
```

Comment:
As mentioned in other [comment](https://github.com/google/site-kit-wp/pull/9779/files#r1860678009) this setting type needed changing to persist ACR metrics on the dashboard once events are lost, and also from them appearing as soon new event is detected.